x-pack/plugins/endpoint/scripts/resolver_generator.ts

```diff
@@ -35,13 +35,13 @@ async function main() {
     eventIndex: {
       alias: 'ei',
       describe: 'index to store events in',
-      default: 'events-endpoint-1',
+      default: 'events-endpoint-default-1',
```

Comment:
🛑 @kqualters-elastic and @marshallmain changed this to an` import`,  which works better I think. I think it was merged upstream?

Reply:
ah, let me take a look

Reply:
no, that was not done

Reply:
Yeah, @kqualters-elastic is right, we're not using an `import` I'll revert this to `events-endpoint-1` though because until the alert details issue get's refactored: https://github.com/elastic/endpoint-app-team/issues/311 it has to stay as `events-endpoint-1`